src/containers/Questions/questions.jsx

```diff
@@ -0,0 +1,178 @@
+import React from 'react';
+import propTypes from 'prop-types';
+import { Button, message, Input, Progress, Icon, Popconfirm } from 'antd';
+import schema from './questionValidation';
+import './questions.css';
+
+const { TextArea } = Input;
+
+const entryData = [
```

Comment:
I think the file would be more readable if the data was in another file, just a suggestion :)